.gitlab-ci.yml

```diff
@@ -59,8 +59,8 @@ Image_test:
   parallel:
     matrix:
       - RUNNER:
-          - aws/fedora-33-x86_64
-          - aws/rhel-8-x86_64
+          - openstack/fedora-33-x86_64
+          - openstack/rhel-8-x86_64
```

Comment:
Why do we need this change?

Reply:
I was just wondering if it runs faster in Openstack but it seems it does not. I'm going to change it back to aws runners. Also I may have wanted to try if it fixes the Openstack test failure but I guess that's just some credentials missing.

Reply:
One of our priorities is to move as many jobs as possible out of Openstack.